src/controllers/v1/usersControllers.ts

```diff
@@ -0,0 +1,67 @@
+import HttpError from '../../errorModel';
+import { Request, Response, NextFunction } from 'express';
+import pool from '../../db';
+import bcrypt from 'bcryptjs';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+
+
+// Admin/ Create User
+// POST /api/v1/users/admin/createUser
+
+export const createUser = async (req: Request, res: Response, next: NextFunction) => {
+
+    try {
+         let { firstname, lastname, email, password, gender, jobrole, department, address} = req.body;
+
+         if(!firstname || !lastname || !email || !password || !gender || !jobrole || !department || !address) {
+            return next(new HttpError('All fields are required', 422));
+         }
+
+         const newEmail = email.toLowerCase();
+
+         const emailExists = await pool.query('SELECT * FROM "userModel" WHERE email = $1', [newEmail]);
```

Comment:
This looks nice,
But it can be better.
Create a query folder, just like how you have controllers, 
You can call it queries, then inside the queries, you will have something like user.queries.ts or what ever for all queries relating to users. 
That way, your seprate the query from the controller, and only import the query and utilize it where necessary.
It also helps you prevent duplication of queries.